app/src/pages/inside/profilePage/modals/deleteAccountFeedbackModal/deleteAccountFeedbackModal.jsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { defineMessages, useIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import { ModalLayout, withModal } from 'components/main/modal';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { showModalAction } from 'controllers/modal';
+import { InputCheckbox } from 'components/inputs/inputCheckbox';
+import { Input } from 'components/inputs/input';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import styles from './deleteAccountFeedbackModal.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  header: {
+    id: 'DeleteAccountFeedbackModal.header',
+    defaultMessage: 'Delete account',
+  },
+  description: {
+    id: 'DeleteAccountFeedbackModal.description',
+    defaultMessage: 'Why are you deleting your account?',
+  },
+  continue: {
+    id: 'DeleteAccountFeedbackModal.continue',
+    defaultMessage: 'Continue',
+  },
+  NO_NEEDED: {
+    id: 'DeleteAccountFeedbackModal.NO_NEEDED',
+    defaultMessage: 'The account is no longer needed',
+  },
+  DISSATISFIED: {
+    id: 'DeleteAccountFeedbackModal.DISSATISFIED',
+    defaultMessage: 'Dissatisfied with the service',
+  },
+  ALTERNATIVE: {
+    id: 'DeleteAccountFeedbackModal.ALTERNATIVE',
+    defaultMessage: 'Found a better alternative',
+  },
+  OTHER: {
+    id: 'DeleteAccountFeedbackModal.OTHER',
+    defaultMessage: 'Other',
+  },
+  inputError: {
+    id: 'DeleteAccountFeedbackModal.inputError',
+    defaultMessage: 'The field should have size not more than 128 symbols.',
+  },
+});
+
+const NO_NEEDED = 'NO_NEEDED';
+const DISSATISFIED = 'DISSATISFIED';
+const ALTERNATIVE = 'ALTERNATIVE';
+const OTHER = 'OTHER';
+
+const DeleteAccountFeedback = () => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const [options, setOptions] = useState({
```

Comment:
Could you explain, why you decided to use local state instead of using redux-form?

Reply:
We will need to pass this data to the next modal (deleteAccountModal) after account deleting.

Reply:
We can easily achieve the same results using redux-form value props.
But in this case, I am not absolutely sure that we need to use redux-form.
@AmsterGet, could you please, take a look at this?

Reply:
The main goals of using Redux form in our project from my point of view:
- Simplify the process of forms implementation (all things like handling input, validation, errors drilling and so on are managed by library);
- Unified approach across the project.

So as this component at least require validation, I suggest to use Redux form.
The good example where the Redux form is an overhead is component from this PR - `DeleteAccountModal`.